src/components/20-molecules/dropdown/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@axa-ch/dropdown",
+  "version": "1.0.0-alpha.2",
+  "description": "The dropdown for the AXA patterns library",
+  "author": "Pattern Warriors",
+  "homepage": "https://github.com/axa-ch/patterns-library",
```

Comment:
this should be to the component, please see other

Reply:
done.